documentation/aircraft-commands.md

```diff
@@ -276,6 +277,13 @@ Altitude also accepts an `expedite` or `x` argument which can be used as the las
 
 _Syntax -_ `AAL123 c [alt]` or `AAL123 c [alt] x`
 
+## Fly Present Heading
```

Comment:
This should be a h3 heading

Reply:
Should also be an empty line after headings

Reply:
done